src/js/features/conversations.js

```diff
@@ -127,7 +127,9 @@ Conversations.prototype.newConversation = function(element) {
 
     if (bttv.settings.get('hideConversations')) this.slideUp();
     $(element).find('.header-button-container').children().last().on('click', function() {
-        if (bttv.settings.get('hideConversations') && $('.conversation-window').length === 1) _self.slideDown();
+        setTimeout(function() {
+            if (bttv.settings.get('hideConversations')) _self.slideDown();
```

Comment:
What happened to `$('.conversation-window').length === 1` here? Would removing that without a `setTimeout` fix the original issue earlier?